demos/applications/icm20948.cpp

```diff
@@ -28,31 +43,30 @@ hal::status application(hardware_map& p_map)
 
   hal::print(console, "icm Application Starting...\n\n");
   (void)hal::delay(clock, 200ms);
```

Comment:
```suggestion
  hal::delay(clock, 200ms);
```

You shouldn't need to use void casts to ignore the return of delay. It doesn't have a return value anymore as of libhal/2.0.0